src/views/authors/detail.pug

```diff
@@ -19,7 +19,9 @@ block content
         p
           strong #{t('authors.dateOfBirth')}:
           | #{new Date(author.dateOfBirth).toDateString()}
-
+        p
+          strong #{t('authors.bio')}:
+          | !{author.bio.replace(/\n/g, '<br>')}
         .mt-4
           h3 #{t('authors.songs')}
           if author.songs && author.songs.length > 0
```

Comment:
đoạn ni chỉ hiển thị 1 ít thôi. 
Thêm thẻ a/button "Hiển thị thêm" Để show Modal list songs of author nhỉ